webbpsf/optics.py

```diff
@@ -1469,19 +1478,27 @@ def __init__(self, instrument, **kwargs):
             instrument,
             **kwargs)
 
-        # Polynomial equations fit to defocus model (RMS WFE).
+        # Polynomial equations fit to defocus model. Wavelength-dependent focus 
+        # results should correspond to Zernike coefficients in meters.
         # Fits were performed to the SW and LW optical design focus model 
         # as provided by Randal Telfer. 
         # See plot at https://github.com/spacetelescope/webbpsf/issues/179
         # The relative wavelength dependence of these focus models are very
         # similar for coronagraphic mode in the Zemax optical prescription,
         # so we opt to use the same focus model in both imaging and coronagraphy.
-        cf_scale = -1.09746e7 # convert from mm defocus to meters RMS wfe
-        sw_cf = np.array([-5.169185169, 50.62919436, -201.5444129, 415.9031962,  
-                          -465.9818413, 265.843112, -59.64330811]) / cf_scale
-        lw_cf = np.array([0.175718713, -1.100964635, 0.986462016, 1.641692934]) / cf_scale
-        self.fm_short = np.poly1d(sw_cf)
-        self.fm_long  = np.poly1d(lw_cf)
+        defocus_to_rmswfe = -1.09746e7 # convert from mm defocus to meters (WFE)
+        sw_focus_cf = np.array([-5.169185169, 50.62919436, -201.5444129, 415.9031962,  
+                                -465.9818413, 265.843112, -59.64330811]) / defocus_to_rmswfe
+        lw_focus_cf = np.array([0.175718713, -1.100964635, 0.986462016, 1.641692934]) / defocus_to_rmswfe
+        self.fm_short = np.poly1d(sw_focus_cf)
+        self.fm_long  = np.poly1d(lw_focus_cf)
+        
+        # Coronagraphic tilt offset model
```

Comment:
[change]
Can you please add here a brief comment describing where these models come from, and how they were derived? The text about deriving from Zemax in the comment from the PR would suffice.